integration/bytes-as-base64/message.ts

```diff
@@ -5,7 +5,7 @@ export interface Message {
   data: Uint8Array;
 }
 
-const baseMessage: object = {};
+const baseMessage: object = { data: new Uint8Array() };
```

Comment:
@webmaster128 is `Uint8Array` immutable? I'd assumed it wasn't, and being in the `baseMessage` would mean this this is shared across multiple `Message`s and so they'd end up sharing each other's `data` instance?

Reply:
Nice catch, you're right. Uint8Array is mutable. I forgot about that because we never mutate Uint8Arrays.

Are you okey with turning baseMessage into a function that creates instances on demand because of this? Or do you have a better idea?

Reply:
Cool. Maybe keep `baseMessage` as-is and add a new line  in the various `decode` / `fromJSON` methods that does `message.data = new Uint8Array()`?

Similar to what we do for lists and maps, where there are a few lines at the start of `decode` that initialize them to empty values.

Reply:
@stephenh I updated this PR to remove the mutable `Uint8Array`s and `Buffer`s from all the base objects (like how lists are treated), which I'm not completely sure is what you meant here. I also adjusted the `fromPartial` methods for consistency. Happy to update if any of this is wrong.

Reply:
@willclarktech cool! Fwiw did your changes to `main.ts` not get pushed up? Looking at the diff + the 3rd commit you pushed to the branch, it looks like maybe only the integration test changes got pushed? Or maybe I'm missing it.

Reply:
@stephenh Whoops! I've updated, and actually changed my mind about `fromPartial`, so that's no longer included in this change.